frontend/src/components/SignupForm/SignupForm.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useContext } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './SignupForm.css'
+import { UserContext } from '../../UserContext.js';
+
+const SignupForm = () => {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const { updateUser } = useContext(UserContext);
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      // Make the signup API request
+      const response = await fetch(`http://localhost:3000/users`, {
```

Comment:
same here about `localhost` in production

Reply:
Done